src/components/popover/popover.tsx

```diff
@@ -56,7 +56,13 @@ export default function Popover({ children }: PopoverProps) {
               right: x ?? 16,
             }}
           >
-            <Button className="close" onClick={() => setIsOpen(false)} />
+            <button
+              type="button"
+              className="close"
+              onClick={() => setIsOpen(false)}
```

Comment:
Let's move the arrow function to the handler. It's better not to keep arrow functions related to handlers in JSX.

Reply:
Moved the function out of JSX to its own handler.